src/components/form-authentication-components/Input.tsx

```diff
@@ -0,0 +1,36 @@
+type InputType = {
```

Comment:
fajnie byloby zrobic reuzywalnego inputa z wykorzystaniem React Hook Form, mozemy to zrobic na zajeciach :)